example_app/templates/example_app/blocks/header.html

```diff
@@ -26,18 +26,62 @@
         <nav role="navigation" class="fr-nav" id="fr-navigation" aria-label="Menu principal">
           <ul class="fr-nav__list">
             <li class="fr-nav__item">
-              <a class="fr-nav__link" href="{% url 'index' %}" target="_self">Accueil</a>
+              {% url 'index' as home_url %}
+              <a class="fr-nav__link" href="{{ home_url }}" target="_self"{% if request.path == home_url %} aria-current="true"{% endif %}>Accueil</a>
             </li>
             <li class="fr-nav__item">
-              <a class="fr-nav__link" href="{% url 'tags_index' %}" target="_self">Composants</a>
+              <button class="fr-nav__btn" aria-expanded="false" aria-controls="menu-doc"{% if '/doc-' in request.path %}aria-current="true"{% endif %}>Documentation</button>
```

Comment:
Je ne crois pas que l'espace manquant pose problème, mais au cas où : 
```suggestion
              <button class="fr-nav__btn" aria-expanded="false" aria-controls="menu-doc"{% if '/doc-' in request.path %} aria-current="true"{% endif %}>Documentation</button>
```